server/application/terminal.go

```diff
@@ -318,6 +320,19 @@ func startProcess(k8sClient kubernetes.Interface, cfg *rest.Config, namespace, p
 		return err
 	}
 
+	// Fallback executor is default, unless feature flag is explicitly disabled.
+	// Reuse environment variable for kubectl to disable the feature flag, default is enabled.
+	if !cmdutil.RemoteCommandWebsockets.IsDisabled() {
```

Comment:
documenting that KUBECTL_REMOTE_COMMAND_WEBSOCKETS=false will not execute this logic https://github.com/kubernetes/kubernetes/blob/d06398aac3e378da0c95472cd39713998206e9ff/staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go#L431